make/CreateJmods.gmk

```diff
@@ -69,7 +69,7 @@ ifneq ($(LIBS_DIR), )
         SRC := $(LIBS_DIR), \
         DEST := $(LIBS_DIR_FILTERED), \
         FILES := $(FILES_LIBS), \
-        NAME_MACRO := rename_stripped, \
+        NAME_MACRO := rename_stripped \
```

Comment:
These trailing commas were deliberate and are part of the [Code Conventions for the Build System](https://openjdk.org/groups/build/doc/code-conventions.html) (17, though it's not very clearly stated).